lint/eslint.bzl

```diff
@@ -247,6 +248,9 @@ def lint_eslint_aspect(binary, configs):
                 allow_single_file = True,
                 cfg = "exec",
             ),
+            "_rule_kinds": attr.string_list(
```

Comment:
@alexeagle The bazel documentation indicates that implicit attrs must be labels. Is that documentation out of date or in error?

https://bazel.build/rules/lib/globals/bzl.html#aspect

Reply:
Interesting, I don't recall reading that doc. Apparently it's incorrect - I verified this has the desired behavior.

The real bug is there's no tech writer assigned to the Bazel project. I've been working on fixing it 😉

Reply:
Good thing you tackled it then. I was thinking that restriction preventing passing the string array via context was going to make the implementation messier than I would have liked.

Reply:
I always start from wishful thinking and then backoff in the face of errors :)